src/services/errorMonitor.ts

```diff
@@ -7,7 +7,7 @@ export const errorMonitor = (res: APIResponse) => {
   if (!res.ok && !exceptionalCase) {
     switch (res.status) {
       case 400:
-        window.location.href = "/error400"
+        // window.location.href = "/error400"
```

Comment:
maybe a miss, that this is commented out ?

Reply:
yes, good catch!